simdb/run.py

```diff
@@ -288,7 +290,15 @@ def process_data(v):
     data = {k: process_data(v) for k, v in _current_dataset_data.items()}
 
     with open(os.path.join(_current_dataset, 'DATA'), 'wb') as f:
-        dump(data, f, protocol=-1)
+        try:
+            dump(data, f, protocol=-1)
+        except PicklingError:
+            for kk, vv in data.items():
+                try:
+                    dump({kk: vv}, f, protocol=-1)
+                except PicklingError as e:
+                    print(f'could not pickle "{kk}"')
+                    raise e
```

Comment:
This seems helpful. Can you make a separate PR for it?

Reply:
see #10